iocage_lib/ioc_fetch.py

```diff
@@ -866,6 +866,10 @@ def fetch_update(self, cli=False, uuid=None):
         path = '/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/sbin:'\
                '/usr/local/bin:/root/bin'
         fetch_env = {
+            **{
```

Comment:
We should ensure `http_proxy` as well and we also have `https_proxy`, so we should check that too

Reply:
As far as I read the sources of freebsd-update, it does only consider those variables. IMHO no need to add others if they are not used.

Reply:
If you would like to follow the path: freebsd-update uses fetch to pull the images, fetch uses fetchXGet() for downloading and according to https://www.freebsd.org/cgi/man.cgi?apropos=0&sektion=3&query=fetchXGet&manpath=FreeBSD+7.0-current&format=html only HTTP_PROXY is used, while http_proxy is for compatibility reasons. Not sure if FTP_PROXY should be added though. AFAIK iocage only uses HTTP nowadays.